ast/parser.go

```diff
@@ -1009,6 +996,7 @@ func (p *Parser) parseLiteral() (expr *Expr) {
 		return p.parseEvery()
 	default:
 		s := p.save()
+		// when error out, expr is nil since false is the tok, when { is the tok, expr is false
```

Comment:
Not sure I understand this comment. It would be helpful if an example was added.

Reply:
Sorry for the confusion. I added some comments to track down the error during debugging. I will now remove the comments since this is the existing code with no changes.